ocs_ci/helpers/helpers.py

```diff
@@ -2942,6 +2943,28 @@ def get_mon_pdb():
     return disruptions_allowed, min_available_mon
 
 
+def safe_get_attr(obj, attribute):
+    """
+    Safely retrieve object attributes
```

Comment:
Docstring needs little improvement:
- don't note that it's safe ...
- stress that it tries to get value of given attribute chain/list
- stress that it doesn't throw exception but it returns None in case an attribute doesn't exist